multichain-testing/Makefile

```diff
@@ -106,5 +115,5 @@ wait-for-pods:
 	scripts/pod-readiness.ts
 
 .PHONY: start
-start: install wait-for-pods port-forward fund-provision-pool override-chain-registry register-bank-assets
+start: install wait-for-pods port-forward fund-provision-pool override-chain-registry register-bank-assets create-osmosis-pool
```

Comment:
This will apply to config files, some of which will not care about osmosis swaps. I think it's help to have the `create-osmosis-pool` command, but we shouldn't include it in`start`